contracts/storage/RibbonTreasuryVaultStorage.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+abstract contract RibbonTreasuryVaultStorageV1 {
+    // Logic contract used to price options
+    address public optionsPremiumPricer;
+    // Logic contract used to select strike prices
+    address public strikeSelection;
+    // Premium discount on options we are selling (thousandths place: 000 - 999)
+    uint256 public premiumDiscount;
+    // Current oToken premium
+    uint256 public currentOtokenPremium;
+    // Last round id at which the strike was manually overridden
+    uint16 public lastStrikeOverrideRound;
+    // Price last overridden strike set to
+    uint256 public overriddenStrikePrice;
+    // Auction duration
+    uint256 public auctionDuration;
+    // Auction id of current option
+    uint256 public optionAuctionID;
+}
+
+abstract contract RibbonTreasuryVaultStorageV2 {
```

Comment:
We can just combine all of them into RibbonTreasuryVaultStorageV1 because no treasury vault has been deployed yet